pkg/operator/baremetal_config.go

```diff
@@ -9,7 +12,16 @@ import (
 )
 
 var provisioningGVR = schema.GroupVersionResource{Group: "metal3.io", Resource: "provisionings", Version: "v1alpha1"}
-var baremetalProvisioningCR = "cluster"
+var baremetalProvisioningCR = "provisioning-configuration"
```

Comment:
nit: this should rather be a constant as well

Reply:
Being addressed as part of https://github.com/openshift/machine-api-operator/pull/482.